lib/src/map/flutter_map_state.dart

```diff
@@ -52,11 +52,11 @@ class FlutterMapState extends MapGestureMixin {
           .map((layer) => _createLayer(layer, widget.options.plugins))
           .toList();
       return new GestureDetector(
-        onScaleStart: handleScaleStart,
-        onScaleUpdate: handleScaleUpdate,
-        onScaleEnd: handleScaleEnd,
+        onScaleStart: options.interactive ? handleScaleStart : null,
+        onScaleUpdate: options.interactive ? handleScaleUpdate : null,
```

Comment:
Can you help me understand why this is required? I thought that if `handleScaleStart` is null the map will behave normally. I don't see why `interactive` is required for the API.